src/main/resources/lib/ssb/statistic.ts

```diff
@@ -254,6 +280,59 @@ function prepStatistics(statistics: Array<Content<Statistics>>): Array<Statistic
   return sortByNextRelease(statisticData)
 }
 
+function getStatisticsJobLogInfo(id: string, count: number = 1): Array<DashboardJobInfo> {
+  return withConnection(EVENT_LOG_REPO, EVENT_LOG_BRANCH, (connection) => {
+    const statisticsJobLog: NodeQueryResponse = connection.query({
+      query: `_path LIKE "/jobs/*" AND data.task = "${JobNames.STATISTICS_REFRESH_JOB}" AND data.queryIds = "${id}"`,
+      count,
+      sort: '_ts DESC'
+    })
+    return statisticsJobLog.hits.reduce((res: Array<DashboardJobInfo>, jobRes) => {
+      const jobNode: JobInfoNode | null = connection.get(jobRes.id)
+      if (jobNode) {
+        res.push(prepStatisticsJobLogInfo(jobNode))
+      }
+      return res
+    }, [])
+  })
+}
+
+function prepStatisticsJobLogInfo(jobNode: JobInfoNode): DashboardJobInfo {
+  const jobResult: Array<RefreshDatasetResult> = forceArray(jobNode.data.refreshDataResult || []) as Array<RefreshDatasetResult>
+  jobResult.forEach((datasetResult: RefreshDatasetResult) => {
+    datasetResult.status = i18n.localize({
+      key: datasetResult.status
+    })
+  })
+  return {
+    id: jobNode._id,
+    startTime: jobNode.data.jobStarted,
+    completionTime: jobNode.data.completionTime ? jobNode.data.completionTime : undefined,
+    status: jobNode.data.status,
+    task: jobNode.data.task,
+    message: jobNode.data.message ? jobNode.data.message : '',
+    result: jobResult,
+    user: jobNode.data.user
+  }
+}
+
+// function getEventLogsFromStatisticsJobLog(connection: RepoConnection, jobLogId: string): Array<unknown> {
+//   const jobLog: JobInfoNode = connection.get(jobLogId)
+//   const userLogin: string | undefined = jobLog.data.user?.login
+//   const from: string = jobLog.data.jobStarted
+//   const to: string = jobLog.data.completionTime
+//   const datasetIds: Array<string> = jobLog.data.refreshDataResult as Array<string> || []
+//   return datasetIds.map((id) => {
+//     log.info(`_path LIKE "/queries/${id}/*" AND data.by.login = "${userLogin}" AND range("_ts", instant("${from}"), instant("${to}"))`)
+//     const eventLogsResult: NodeQueryResponse = connection.query({
+//       query: `_path LIKE "/queries/${id}/*" AND data.by.login = "${userLogin}" AND range("_ts", instant("${from}"), instant("${to}"))`,
+//       count: 10,
+//       sort: '_ts DESC'
+//     })
+//     return null
+//   })
+// }
+
```

Comment:
Commented code

Reply:
added note, this is an example on how to get eventlogs for datasource refresh on statistics job log